src/main/java/net/minecraftforge/client/gui/HoverChecker.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016-2019.
```

Comment:
It's 2020 now, I assume this should be updated?

Reply:
Yes but it needs updating repository-wide, otherwise the license checker script will complain.

Reply:
The file containing that is the LICENSE-header.txt at the root; I ran the task to format the license so it was automatically added. Another PR needs to be done that updates both the license header template and all files' header.